src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -840,7 +847,7 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 				// calculate the correct step size and provide it via a parameter.
 				if( AiNodeGetFlt( node, g_stepSizeArnoldString ) == 0.0f )
 				{
-					AiNodeSetFlt( node, g_stepSizeArnoldString, m_stepSize );
+					AiNodeSetFlt( node, g_stepSizeArnoldString, actualStepSize );
```

Comment:
Hmm, so if a procedural specifies its own step_size (as per the comment above), then we lose the step_scale behaviour entirely? I thought our aim was to make it so that step_scale was always taken into account, so this seems unfortunate.